### ISSUE TITLE
Error executing process 'cnv_spectre:makeReport (1)'

### ISSUE THREAD

**Issue by cihaterdogan** [epi2me-labs/wf-human-variation#165]
### Operating System

Other Linux (please specify below)

### Other Linux

Red Hat Enterprise 8.x

### Workflow Version

v2.0.0

### Workflow Execution

Other (please describe)

### CLI command run

export NXF_SINGULARITY_CACHEDIR=Path/.singularity
export SINGULARITY_TMPDIR=Path/.singularity
export NXF_HOME=Path

OUTPUT=Su715_BTRC_215_AUD_F_results

./nextflow run epi2me-labs/wf-human-variation \
	-w ${OUTPUT}/workspace \
	-profile singularity \
	--bam 'ONT_000_COGA_DNA150_Dec2023/Su715_BTRC_215_AUD_F_simplex_sup_5mc5hmc_m6A_unaligned.bam' \
	--basecaller_cfg 'dna_r10.4.1_e8.2_400bps_sup@v4.3.0' \
	--mod \
	--ref 'hg38/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna' \
	--sample_name 'Su715_BTRC_215_AUD_F' \
	--sv \
	--snp \
	--str \
	--cnv \
	--phased \
	--sex female \
	--threads 32 \
	--bam_min_coverage 5 \
	--out_dir ${OUTPUT}

### Workflow Execution - CLI Execution Profile

singularity

### What happened?

I was running the wf-human-variation pipeline as a job (with Slurm) on my institution's clusters with the following input

#!/bin/bash
#SBATCH --nodes=1
#SBATCH --ntasks-per-node=1
#SBATCH --cpus-per-task=32
#SBATCH --time=0-24:00:00
#SBATCH --mem=200gb

module load singularity/3.8.3
module load java/17.0.0

OUTPUT=Su715_BTRC_215_AUD_F_results

./nextflow run epi2me-labs/wf-human-variation \
	-w ${OUTPUT}/workspace \
	-profile singularity \
	--bam 'ONT_000_COGA_DNA150_Dec2023/Su715_BTRC_215_AUD_F_simplex_sup_5mc5hmc_m6A_unaligned.bam' \
	--basecaller_cfg 'dna_r10.4.1_e8.2_400bps_sup@v4.3.0' \
	--mod \
	--ref 'hg38/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna' \
	--sample_name 'Su715_BTRC_215_AUD_F' \
	--sv \
	--snp \
	--str \
	--cnv \
	--phased \
	--sex female \
	--threads 32 \
	--bam_min_coverage 5 \
	--out_dir ${OUTPUT}

I successfully analyzed five samples using the same parameters with this tool but got the following error in one sample.

ERROR ~ Error executing process > 'cnv_spectre:makeReport (1)'

### Relevant log output

```shell
executor >  local (2485)
[67/bdb901] process > index_ref_fai (1)              [100%] 1 of 1 ✔
[06/53b25d] process > bam_ingress:samtools_index (1) [100%] 1 of 1 ✔
[cd/e579e9] process > bam_ingress:check_for_align... [100%] 1 of 1 ✔
[-        ] process > bam_ingress:cram_to_bam        -
[54/3a4cdf] process > bam_ingress:minimap2_alignm... [100%] 1 of 1 ✔
[1d/b0e0ac] process > getGenome (1)                  [100%] 1 of 1 ✔
[b4/6aee5b] process > cram_cache (1)                 [100%] 1 of 1 ✔
[49/1a54f4] process > getAllChromosomesBed (1)       [100%] 1 of 1 ✔
[92/6301fa] process > mosdepth_input (1)             [100%] 1 of 1 ✔
[79/291120] process > getVersions                    [100%] 1 of 1 ✔
[23/f8fc9d] process > getParams                      [100%] 1 of 1 ✔
[23/57a588] process > get_coverage (1)               [100%] 1 of 1 ✔
[56/6a0039] process > readStats (1)                  [100%] 1 of 1 ✔
[d4/3af68e] process > makeAlignmentReport            [100%] 1 of 1 ✔
[-        ] process > failedQCReport                 -
[13/81b87d] process > lookup_clair3_model (1)        [100%] 1 of 1 ✔
[74/9eebd7] process > snp:make_chunks (1)            [100%] 1 of 1 ✔
[fe/f4225b] process > snp:pileup_variants (618)      [100%] 632 of 632 ✔
[c4/b594fb] process > snp:aggregate_pileup_varian... [100%] 1 of 1 ✔
[be/a8b2ce] process > snp:select_het_snps (2)        [100%] 25 of 25 ✔
[11/163751] process > snp:phase_contig (25)          [100%] 25 of 25 ✔
[43/1178ae] process > snp:get_qual_filter (1)        [100%] 1 of 1 ✔
[91/029166] process > snp:create_candidates (2)      [100%] 25 of 25 ✔
[9a/1f4e7b] process > snp:evaluate_candidates (1557) [100%] 1561 of 1561 ✔
[11/64c22d] process > snp:aggregate_full_align_va... [100%] 1 of 1 ✔
[7d/a980de] process > snp:merge_pileup_and_full_v... [100%] 25 of 25 ✔
[ae/725fc9] process > snp:post_clair_phase_contig... [100%] 25 of 25 ✔
[35/7f864f] process > snp:post_clair_contig_haplo... [ 88%] 22 of 25
[-        ] process > snp:cat_haplotagged_contigs    -
[38/4eb332] process > snp:aggregate_all_variants (1) [100%] 1 of 1 ✔
[-        ] process > sv:variantCall:sniffles2       -
[-        ] process > sv:variantCall:filterCalls     -
[-        ] process > sv:variantCall:sortVCF         -
[-        ] process > sv:annotate_sv_vcf             -
[a1/166cb4] process > sv:runReport:getVersions       [100%] 1 of 1 ✔
[22/3ce628] process > sv:runReport:getParams         [100%] 1 of 1 ✔
[-        ] process > sv:runReport:report            -
[-        ] process > output_sv                      -
[-        ] process > refine_with_sv                 -
[-        ] process > annotate_snp_vcf               -
[-        ] process > concat_snp_vcfs                -
[-        ] process > sift_clinvar_snp_vcf           -
[-        ] process > vcfStats                       -
[a6/abcd62] process > report_snp:getVersions         [100%] 1 of 1 ✔
[e0/fd49d4] process > report_snp:getParams           [100%] 1 of 1 ✔
[-        ] process > report_snp:makeReport          -
[-        ] process > output_snp                     -
[-        ] process > validate_modbam                -
[-        ] process > mod:modkit_phase               -
[95/ee37bf] process > cnv_spectre:mosdepth (1)       [100%] 1 of 1 ✔
[eb/43bcbb] process > cnv_spectre:callCNV (1)        [100%] 1 of 1 ✔
[9c/d625c5] process > cnv_spectre:bgzip_and_index... [100%] 1 of 1 ✔
[16/65101c] process > cnv_spectre:annotate_vcf (1)   [  0%] 0 of 1
[aa/af6a34] process > cnv_spectre:getVersions        [100%] 1 of 1 ✔
[9c/4d11b2] process > cnv_spectre:add_snp_tools_t... [100%] 1 of 1 ✔
[44/808749] process > cnv_spectre:getParams          [100%] 1 of 1 ✔
[c9/168bd1] process > cnv_spectre:makeReport (1)     [  0%] 0 of 1
[-        ] process > output_cnv                     -
[44/aa7757] process > str:call_str (22)              [100%] 22 of 22
[47/7f0246] process > str:annotate_repeat_expansi... [100%] 17 of 17
[a2/b4cb86] process > str:getVersions                [100%] 1 of 1 ✔
[df/3ce05f] process > str:getParams                  [100%] 1 of 1 ✔
[75/054f75] process > str:bam_region_filter (22)     [100%] 22 of 22
[85/e51e22] process > str:bam_read_filter (17)       [100%] 17 of 17
[be/c58d7f] process > str:generate_str_content (16)  [ 94%] 16 of 17
[-        ] process > str:concat_str_vcfs            -
[-        ] process > str:merge_tsv                  -
[-        ] process > str:make_report                -
[-        ] process > output_str                     -
[-        ] process > phasing:phase_all              -
[-        ] process > phasing:concat_phased_vcfs     -
[-        ] process > phasing:haploblocks_joint      -
[d3/12770f] process > configure_jbrowse (1)          [100%] 1 of 1 ✔
[07/82d5e1] process > publish_artifact (13)          [100%] 13 of 13
ERROR ~ Error executing process > 'cnv_spectre:makeReport (1)'

Caused by:
  Process `cnv_spectre:makeReport (1)` terminated with an error exit status (1)

Command executed:

  workflow-glue report_cnv_spectre             --sample_id Su715_BTRC_215_AUD_F             --params params.json             --versions versions             --cnv_bed Su715_BTRC_215_AUD_F_cnv.bed             --chr_codes chr1,1,chr2,2,chr3,3,chr4,4,chr5,5,chr6,6,chr7,7,chr8,8,chr9,9,chr10,10,chr11,11,chr12,12,chr13,13,chr14,14,chr15,15,chr16,16,chr17,17,chr18,18,chr19,19,chr20,20,chr21,21,chr22,22,chrX,X,chrY,Y,chrM,M,chrMT,MT             -o Su715_BTRC_215_AUD_F.wf-human-cnv-report.html

Command exit status:
  1

Command output:
  (empty)

Command error:
  [15:35:43 - workflow_glue] Starting entrypoint.
  Traceback (most recent call last):
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow-glue", line 7, in <module>
      cli()
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow_glue/__init__.py", line 72, in cli
      args.func(args)
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow_glue/report_cnv_spectre.py", line 121, in main
      cnv_df = pd.read_csv(args.cnv_bed, delim_whitespace=True, header=None)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/util/_decorators.py", line 211, in wrapper
      return func(*args, **kwargs)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/util/_decorators.py", line 331, in wrapper
      return func(*args, **kwargs)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
      return _read(filepath_or_buffer, kwds)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 605, in _read
      parser = TextFileReader(filepath_or_buffer, **kwds)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
      self._engine = self._make_engine(f, self.engine)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1753, in _make_engine
      return mapping[engine](f, **self.options)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 79, in __init__
      self._reader = parsers.TextReader(src, **kwds)
    File "pandas/_libs/parsers.pyx", line 554, in pandas._libs.parsers.TextReader.__cinit__
  pandas.errors.EmptyDataError: No columns to parse from file

Work dir:
  /N/scratch/cerdogan/nextflow_br200/Su715_BTRC_215_AUD_F_results/workspace/c9/168bd1048af1ef52c932fd3328bd53

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

 -- Check '.nextflow.log' file for details

executor >  local (2485)
[67/bdb901] process > index_ref_fai (1)              [100%] 1 of 1 ✔
[06/53b25d] process > bam_ingress:samtools_index (1) [100%] 1 of 1 ✔
[cd/e579e9] process > bam_ingress:check_for_align... [100%] 1 of 1 ✔
[-        ] process > bam_ingress:cram_to_bam        -
[54/3a4cdf] process > bam_ingress:minimap2_alignm... [100%] 1 of 1 ✔
[1d/b0e0ac] process > getGenome (1)                  [100%] 1 of 1 ✔
[b4/6aee5b] process > cram_cache (1)                 [100%] 1 of 1 ✔
[49/1a54f4] process > getAllChromosomesBed (1)       [100%] 1 of 1 ✔
[92/6301fa] process > mosdepth_input (1)             [100%] 1 of 1 ✔
[79/291120] process > getVersions                    [100%] 1 of 1 ✔
[23/f8fc9d] process > getParams                      [100%] 1 of 1 ✔
[23/57a588] process > get_coverage (1)               [100%] 1 of 1 ✔
[56/6a0039] process > readStats (1)                  [100%] 1 of 1 ✔
[d4/3af68e] process > makeAlignmentReport            [100%] 1 of 1 ✔
[-        ] process > failedQCReport                 -
[13/81b87d] process > lookup_clair3_model (1)        [100%] 1 of 1 ✔
[74/9eebd7] process > snp:make_chunks (1)            [100%] 1 of 1 ✔
[fe/f4225b] process > snp:pileup_variants (618)      [100%] 632 of 632 ✔
[c4/b594fb] process > snp:aggregate_pileup_varian... [100%] 1 of 1 ✔
[be/a8b2ce] process > snp:select_het_snps (2)        [100%] 25 of 25 ✔
[11/163751] process > snp:phase_contig (25)          [100%] 25 of 25 ✔
[43/1178ae] process > snp:get_qual_filter (1)        [100%] 1 of 1 ✔
[91/029166] process > snp:create_candidates (2)      [100%] 25 of 25 ✔
[9a/1f4e7b] process > snp:evaluate_candidates (1557) [100%] 1561 of 1561 ✔
[11/64c22d] process > snp:aggregate_full_align_va... [100%] 1 of 1 ✔
[7d/a980de] process > snp:merge_pileup_and_full_v... [100%] 25 of 25 ✔
[ae/725fc9] process > snp:post_clair_phase_contig... [100%] 25 of 25 ✔
[35/7f864f] process > snp:post_clair_contig_haplo... [ 88%] 22 of 25
[-        ] process > snp:cat_haplotagged_contigs    -
[38/4eb332] process > snp:aggregate_all_variants (1) [100%] 1 of 1 ✔
[-        ] process > sv:variantCall:sniffles2       -
[-        ] process > sv:variantCall:filterCalls     -
[-        ] process > sv:variantCall:sortVCF         -
[-        ] process > sv:annotate_sv_vcf             -
[a1/166cb4] process > sv:runReport:getVersions       [100%] 1 of 1 ✔
[22/3ce628] process > sv:runReport:getParams         [100%] 1 of 1 ✔
[-        ] process > sv:runReport:report            -
[-        ] process > output_sv                      -
[-        ] process > refine_with_sv                 -
[-        ] process > annotate_snp_vcf               -
[-        ] process > concat_snp_vcfs                -
[-        ] process > sift_clinvar_snp_vcf           -
[-        ] process > vcfStats                       -
[a6/abcd62] process > report_snp:getVersions         [100%] 1 of 1 ✔
[e0/fd49d4] process > report_snp:getParams           [100%] 1 of 1 ✔
[-        ] process > report_snp:makeReport          -
[-        ] process > output_snp                     -
[-        ] process > validate_modbam                -
[-        ] process > mod:modkit_phase               -
[95/ee37bf] process > cnv_spectre:mosdepth (1)       [100%] 1 of 1 ✔
[eb/43bcbb] process > cnv_spectre:callCNV (1)        [100%] 1 of 1 ✔
[9c/d625c5] process > cnv_spectre:bgzip_and_index... [100%] 1 of 1 ✔
[16/65101c] process > cnv_spectre:annotate_vcf (1)   [  0%] 0 of 1
[aa/af6a34] process > cnv_spectre:getVersions        [100%] 1 of 1 ✔
[9c/4d11b2] process > cnv_spectre:add_snp_tools_t... [100%] 1 of 1 ✔
[44/808749] process > cnv_spectre:getParams          [100%] 1 of 1 ✔
[c9/168bd1] process > cnv_spectre:makeReport (1)     [100%] 1 of 1, failed: 1 ✘
[-        ] process > output_cnv                     -
[44/aa7757] process > str:call_str (22)              [100%] 22 of 22
[47/7f0246] process > str:annotate_repeat_expansi... [100%] 17 of 17
[a2/b4cb86] process > str:getVersions                [100%] 1 of 1 ✔
[df/3ce05f] process > str:getParams                  [100%] 1 of 1 ✔
[75/054f75] process > str:bam_region_filter (22)     [100%] 22 of 22
[85/e51e22] process > str:bam_read_filter (17)       [100%] 17 of 17
[be/c58d7f] process > str:generate_str_content (16)  [ 94%] 16 of 17
[-        ] process > str:concat_str_vcfs            -
[-        ] process > str:merge_tsv                  -
[-        ] process > str:make_report                -
[-        ] process > output_str                     -
[-        ] process > phasing:phase_all              -
[-        ] process > phasing:concat_phased_vcfs     -
[-        ] process > phasing:haploblocks_joint      -
[d3/12770f] process > configure_jbrowse (1)          [100%] 1 of 1 ✔
[07/82d5e1] process > publish_artifact (13)          [100%] 13 of 13
ERROR ~ Error executing process > 'cnv_spectre:makeReport (1)'

Caused by:
  Process `cnv_spectre:makeReport (1)` terminated with an error exit status (1)

Command executed:

  workflow-glue report_cnv_spectre             --sample_id Su715_BTRC_215_AUD_F             --params params.json             --versions versions             --cnv_bed Su715_BTRC_215_AUD_F_cnv.bed             --chr_codes chr1,1,chr2,2,chr3,3,chr4,4,chr5,5,chr6,6,chr7,7,chr8,8,chr9,9,chr10,10,chr11,11,chr12,12,chr13,13,chr14,14,chr15,15,chr16,16,chr17,17,chr18,18,chr19,19,chr20,20,chr21,21,chr22,22,chrX,X,chrY,Y,chrM,M,chrMT,MT             -o Su715_BTRC_215_AUD_F.wf-human-cnv-report.html

Command exit status:
  1

Command output:
  (empty)

Command error:
  [15:35:43 - workflow_glue] Starting entrypoint.
  Traceback (most recent call last):
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow-glue", line 7, in <module>
      cli()
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow_glue/__init__.py", line 72, in cli
      args.func(args)
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow_glue/report_cnv_spectre.py", line 121, in main
      cnv_df = pd.read_csv(args.cnv_bed, delim_whitespace=True, header=None)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/util/_decorators.py", line 211, in wrapper
      return func(*args, **kwargs)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/util/_decorators.py", line 331, in wrapper
      return func(*args, **kwargs)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
      return _read(filepath_or_buffer, kwds)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 605, in _read
      parser = TextFileReader(filepath_or_buffer, **kwds)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
      self._engine = self._make_engine(f, self.engine)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1753, in _make_engine
      return mapping[engine](f, **self.options)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 79, in __init__
      self._reader = parsers.TextReader(src, **kwds)
    File "pandas/_libs/parsers.pyx", line 554, in pandas._libs.parsers.TextReader.__cinit__
  pandas.errors.EmptyDataError: No columns to parse from file

Work dir:
  /N/scratch/cerdogan/nextflow_br200/Su715_BTRC_215_AUD_F_results/workspace/c9/168bd1048af1ef52c932fd3328bd53

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

 -- Check '.nextflow.log' file for details

executor >  local (2485)
[67/bdb901] process > index_ref_fai (1)              [100%] 1 of 1 ✔
[06/53b25d] process > bam_ingress:samtools_index (1) [100%] 1 of 1 ✔
[cd/e579e9] process > bam_ingress:check_for_align... [100%] 1 of 1 ✔
[-        ] process > bam_ingress:cram_to_bam        -
[54/3a4cdf] process > bam_ingress:minimap2_alignm... [100%] 1 of 1 ✔
[1d/b0e0ac] process > getGenome (1)                  [100%] 1 of 1 ✔
[b4/6aee5b] process > cram_cache (1)                 [100%] 1 of 1 ✔
[49/1a54f4] process > getAllChromosomesBed (1)       [100%] 1 of 1 ✔
[92/6301fa] process > mosdepth_input (1)             [100%] 1 of 1 ✔
[79/291120] process > getVersions                    [100%] 1 of 1 ✔
[23/f8fc9d] process > getParams                      [100%] 1 of 1 ✔
[23/57a588] process > get_coverage (1)               [100%] 1 of 1 ✔
[56/6a0039] process > readStats (1)                  [100%] 1 of 1 ✔
[d4/3af68e] process > makeAlignmentReport            [100%] 1 of 1 ✔
[-        ] process > failedQCReport                 -
[13/81b87d] process > lookup_clair3_model (1)        [100%] 1 of 1 ✔
[74/9eebd7] process > snp:make_chunks (1)            [100%] 1 of 1 ✔
[fe/f4225b] process > snp:pileup_variants (618)      [100%] 632 of 632 ✔
[c4/b594fb] process > snp:aggregate_pileup_varian... [100%] 1 of 1 ✔
[be/a8b2ce] process > snp:select_het_snps (2)        [100%] 25 of 25 ✔
[11/163751] process > snp:phase_contig (25)          [100%] 25 of 25 ✔
[43/1178ae] process > snp:get_qual_filter (1)        [100%] 1 of 1 ✔
[91/029166] process > snp:create_candidates (2)      [100%] 25 of 25 ✔
[9a/1f4e7b] process > snp:evaluate_candidates (1557) [100%] 1561 of 1561 ✔
[11/64c22d] process > snp:aggregate_full_align_va... [100%] 1 of 1 ✔
[7d/a980de] process > snp:merge_pileup_and_full_v... [100%] 25 of 25 ✔
[ae/725fc9] process > snp:post_clair_phase_contig... [100%] 25 of 25 ✔
[35/7f864f] process > snp:post_clair_contig_haplo... [ 88%] 22 of 25
[-        ] process > snp:cat_haplotagged_contigs    -
[38/4eb332] process > snp:aggregate_all_variants (1) [100%] 1 of 1 ✔
[-        ] process > sv:variantCall:sniffles2       -
[-        ] process > sv:variantCall:filterCalls     -
[-        ] process > sv:variantCall:sortVCF         -
[-        ] process > sv:annotate_sv_vcf             -
[a1/166cb4] process > sv:runReport:getVersions       [100%] 1 of 1 ✔
[22/3ce628] process > sv:runReport:getParams         [100%] 1 of 1 ✔
[-        ] process > sv:runReport:report            -
[-        ] process > output_sv                      -
[-        ] process > refine_with_sv                 -
[-        ] process > annotate_snp_vcf               -
[-        ] process > concat_snp_vcfs                -
[-        ] process > sift_clinvar_snp_vcf           -
[-        ] process > vcfStats                       -
[a6/abcd62] process > report_snp:getVersions         [100%] 1 of 1 ✔
[e0/fd49d4] process > report_snp:getParams           [100%] 1 of 1 ✔
[-        ] process > report_snp:makeReport          -
[-        ] process > output_snp                     -
[-        ] process > validate_modbam                -
[-        ] process > mod:modkit_phase               -
[95/ee37bf] process > cnv_spectre:mosdepth (1)       [100%] 1 of 1 ✔
[eb/43bcbb] process > cnv_spectre:callCNV (1)        [100%] 1 of 1 ✔
[9c/d625c5] process > cnv_spectre:bgzip_and_index... [100%] 1 of 1 ✔
[16/65101c] process > cnv_spectre:annotate_vcf (1)   [  0%] 0 of 1
[aa/af6a34] process > cnv_spectre:getVersions        [100%] 1 of 1 ✔
[9c/4d11b2] process > cnv_spectre:add_snp_tools_t... [100%] 1 of 1 ✔
[44/808749] process > cnv_spectre:getParams          [100%] 1 of 1 ✔
[c9/168bd1] process > cnv_spectre:makeReport (1)     [100%] 1 of 1, failed: 1 ✘
[-        ] process > output_cnv                     -
[44/aa7757] process > str:call_str (22)              [100%] 22 of 22
[47/7f0246] process > str:annotate_repeat_expansi... [100%] 17 of 17
[a2/b4cb86] process > str:getVersions                [100%] 1 of 1 ✔
[df/3ce05f] process > str:getParams                  [100%] 1 of 1 ✔
[75/054f75] process > str:bam_region_filter (22)     [100%] 22 of 22
[85/e51e22] process > str:bam_read_filter (17)       [100%] 17 of 17
[be/c58d7f] process > str:generate_str_content (16)  [ 94%] 16 of 17
[-        ] process > str:concat_str_vcfs            -
[-        ] process > str:merge_tsv                  -
[-        ] process > str:make_report                -
[-        ] process > output_str                     -
[-        ] process > phasing:phase_all              -
[-        ] process > phasing:concat_phased_vcfs     -
[-        ] process > phasing:haploblocks_joint      -
[d3/12770f] process > configure_jbrowse (1)          [100%] 1 of 1 ✔
[07/82d5e1] process > publish_artifact (13)          [100%] 13 of 13
ERROR ~ Error executing process > 'cnv_spectre:makeReport (1)'

Caused by:
  Process `cnv_spectre:makeReport (1)` terminated with an error exit status (1)

Command executed:

  workflow-glue report_cnv_spectre             --sample_id Su715_BTRC_215_AUD_F             --params params.json             --versions versions             --cnv_bed Su715_BTRC_215_AUD_F_cnv.bed             --chr_codes chr1,1,chr2,2,chr3,3,chr4,4,chr5,5,chr6,6,chr7,7,chr8,8,chr9,9,chr10,10,chr11,11,chr12,12,chr13,13,chr14,14,chr15,15,chr16,16,chr17,17,chr18,18,chr19,19,chr20,20,chr21,21,chr22,22,chrX,X,chrY,Y,chrM,M,chrMT,MT             -o Su715_BTRC_215_AUD_F.wf-human-cnv-report.html

Command exit status:
  1

Command output:
  (empty)

Command error:
  [15:35:43 - workflow_glue] Starting entrypoint.
  Traceback (most recent call last):
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow-glue", line 7, in <module>
      cli()
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow_glue/__init__.py", line 72, in cli
      args.func(args)
    File "/N/scratch/cerdogan/nextflow_br200/assets/epi2me-labs/wf-human-variation/bin/workflow_glue/report_cnv_spectre.py", line 121, in main
      cnv_df = pd.read_csv(args.cnv_bed, delim_whitespace=True, header=None)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/util/_decorators.py", line 211, in wrapper
      return func(*args, **kwargs)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/util/_decorators.py", line 331, in wrapper
      return func(*args, **kwargs)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
      return _read(filepath_or_buffer, kwds)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 605, in _read
      parser = TextFileReader(filepath_or_buffer, **kwds)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
      self._engine = self._make_engine(f, self.engine)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1753, in _make_engine
      return mapping[engine](f, **self.options)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 79, in __init__
      self._reader = parsers.TextReader(src, **kwds)
    File "pandas/_libs/parsers.pyx", line 554, in pandas._libs.parsers.TextReader.__cinit__
  pandas.errors.EmptyDataError: No columns to parse from file

Work dir:
  /N/scratch/cerdogan/nextflow_br200/Su715_BTRC_215_AUD_F_results/workspace/c9/168bd1048af1ef52c932fd3328bd53

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

 -- Check '.nextflow.log' file for details
WARN: Killing running tasks (5)
```

### Were you able to successfully run the latest version of the workflow with the demo data?

yes

**Comment by cihaterdogan** [comment]
Please also find the log file here
[nextflow.log](https://github.com/epi2me-labs/wf-human-variation/files/14728277/nextflow.log)

**Comment by vlshesketh** [comment]
Hi @cihaterdogan, thank you for reporting this. It looks as if the reporting script is not dealing with the case when no CNVs are detected. I will update here when we have a fix for this.

**Comment by vlshesketh** [comment]
Hi @cihaterdogan, this has now been fixed in [v2.0.0](https://github.com/epi2me-labs/wf-human-variation/releases/tag/v2.2.0).
